package.json

```diff
@@ -23,37 +24,39 @@
   "author": "GoDaddy.com Operating Company LLC",
   "license": "MIT",
   "dependencies": {
-    "async": "~2.1.5",
-    "bffs": "^6.0.1",
-    "bl": "~1.2.1",
-    "booting": "~1.0.0",
-    "datastar": "^1.7.0",
+    "async": "^3.1.0",
+    "aws-liveness": "^1.1.0",
+    "aws-sdk": "^2.569.0",
+    "bffs": "^7.0.0",
+    "bl": "^4.0.0",
+    "booting": "^1.1.0",
+    "dynamodb-x": "^1.2.3",
     "failure": "^1.1.1",
     "millisecond": "~0.1.2",
-    "nconf": "~0.8.4",
+    "nconf": "^0.10.0",
     "nsq-stream": "^2.1.1",
     "nsq.js-k8": "^1.2.0",
-    "one-time": "0.0.4",
-    "parallel-transform": "~1.1.0",
+    "one-time": "^1.0.0",
+    "parallel-transform": "^1.2.0",
     "pkgcloud": "^2.1.1",
     "retryme": "^1.0.0",
     "rip-out": "^1.0.0",
     "slay-config": "~2.3.0",
-    "uuid": "~3.0.1",
-    "warehouse-models": "^5.6.0",
+    "uuid": "^3.3.3",
+    "warehouse-models": "^6.0.0",
     "winston": "~2.3.1",
```

Comment:
We should make issues to get this updated across the board if we don't already have them.

Reply:
Yeah these should def be separate issues, since updating winston will fail some internal `bffs` code as the instanced is passed along? I'll make one to check and update slay and winston across the board for warehouse repo's. Those two dependencies are basically the only two that couldn't be mass updated.